project_stage_state/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
```

Comment:
Please split readme as per https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

Reply:
@alexey-pelykh , Could you please check it now?